retrieval_service/spanner.tests.cloudbuild.yaml

```diff
@@ -40,6 +40,17 @@ steps:
         #!/usr/bin/env bash
         python -m pytest datastore/providers/spanner_gsql_test.py
 
+  - id: Run Spanner with postgres dialect integration tests
```

Comment:
Can we have a separate test `spanner-pg.tests.cloudbuild.yaml` for this. And rename the existing one to `spanner-gsql.tests.cloudbuild.yaml`.